Modelica_DeviceDrivers/Resources/Include/MDDIIO.h

```diff
@@ -0,0 +1,78 @@
+/** Linux Comedi DAQ support (header-only library).
+ *
+ * @file
+ * @author      bernhard-thiele
+ * @since       2012-06-26
+ * @copyright see accompanying file LICENSE_Modelica_DeviceDrivers.txt
+ */
+
+#ifndef MDDIIO_H_
+#define MDDIIO_H_
+
+#include <stdlib.h>
+#include <../thirdParty/libiio/iio.h>
+
+#include "ModelicaUtilities.h"
+
+#define IIO_ENSURE(expr, message, context) { \
+	if (!(expr)) { \
+		(void) ModelicaFormatError("%s %s (%s:%d)\n", message, context, __FILE__, __LINE__); \
+	} \
+}
+
+// Streaming devices
+
+static struct iio_buffer  *rxbuf;
+static struct iio_channel **channels;
+static unsigned int channel_count;
```

Comment:
Why iis it needed? And why static storage class?

Reply:
Well...I used the Comedi code as template, that was a leftover. Fixed in 0663e4c1ef5127f5a246c03cefddb3871f90dc9a